src/test/java/seedu/address/model/person/UniquePersonListTest.java

```diff
@@ -162,6 +162,17 @@ public void setPersons_listWithDuplicatePersons_throwsDuplicatePersonException()
         assertThrows(DuplicatePersonException.class, () -> uniquePersonList.setPersons(listWithDuplicatePersons));
     }
 
+    @Test
+    public void getPersonWithNric_existingPerson_person() {
```

Comment:
Just a nit, maybe getPersonWithNric_personWithNricExists_personReturned() will be clearer?